### DISCUSSION TITLE
Noah-MP Docker Container Setup (Windows): Regression Test Failed

### DISCUSSION THREAD

**Discussion by leiapauline** [NCAR/hrldas#184] [Q&A] [answered]
Hello. I am following the tutorial to setup NoahMP using the Docker container on Windows. I was able to download the UFS/Noah-MP Component Model but there was an error during the test. Any guidance would be greatly appreciated.

Sincerely,
Leia Pauline

Please see below for the error received,
`UFS_tasks         : 6`
`FAILED TESTS:`
`001 datm_cdeps_lnd_era5_gnu failed in check_result`
`REGRESSION TEST FAILED`
`Elapsed time: 00h:03m:41s. Have a nice day!`
`rt.sh finished`,

and a snippet of the log_local file,
`baseline dir = /opt/RT/NEMSfv3gfs/develop-20231003/datm_cdeps_lnd_era5_gnu`
`working dir  = /opt/scratch/root/FV3_RT/rt_575/datm_cdeps_lnd_era5_gnu`
`Checking test 001 datm_cdeps_lnd_era5_gnu results ....`
` Comparing ufs.cpld.lnd.out.2011-01-01-43200.tile1.nc .........NOT OK`
` Comparing ufs.cpld.lnd.out.2011-01-01-43200.tile2.nc .........NOT OK`
` Comparing ufs.cpld.lnd.out.2011-01-01-43200.tile3.nc .........NOT OK`
` Comparing ufs.cpld.lnd.out.2011-01-01-43200.tile4.nc .........NOT OK`
` Comparing ufs.cpld.lnd.out.2011-01-01-43200.tile5.nc .........NOT OK`
` Comparing ufs.cpld.lnd.out.2011-01-01-43200.tile6.nc .........NOT OK`
`Test 001 datm_cdeps_lnd_era5_gnu FAIL`.

**Comment by cenlinhe** [comment]
@uturuncoglu could you please answer this queston?

**Comment by uturuncoglu** [comment, marked as answer]
@leiapauline Since we are using container approach, it is possible to have issue with the previously generated baseline. The baseline might be generated with different host machine which has different cpu and architecture. I suggest you to create baseline from scratch with an additional argument to rt.sh. You just need to add -c to the existing command and this will create new baseline for you. Then, if you want you could copy this to the original location and run rt.sh without -c. This time baseline check will probably pass. Let me know if you still have issue.

**Reply by leiapauline** [reply]
After cloning the git folder, I went to the tests directory and ran rt.sh with the additional argument.
`cd ufs-weather-model/tests`
`./rt.sh -c -a debug -k -n datm_cdeps_lnd_era5 gnu`
The regression test was successful. I will proceed with the rest of the tutorial. Thanks for your time.
